Makefile.defs

```diff
@@ -447,28 +441,28 @@ endif
 ifndef DARWIN
 ifndef ANDROID
 ifndef WIN32
-    # We're a glibc system... Find the actual libc used by our TC, and check if it's >= 2.22...
-    # We only care about that on non-emulated builds (i.e., arm), because compat symbols may be pinned
-    # to different versions on different arches, and we only account for arm in our patches.
-    ifndef EMULATE_READER
-        GLIBC_FILE=$(basename $(notdir $(realpath $(shell PATH='$(PATH)' $(CC) -print-file-name=libc.so.6))))
-        GLIBC_VER=$(shell echo $(GLIBC_FILE) | cut -f2 -d'-')
-        GLIBC_2_22_SORT=$(shell echo -e "$(GLIBC_VER)\n2.22" | sort -V | head -n1)
-        ifeq ($(GLIBC_2_22_SORT), 2.22)
-            ARM_GLIBC_GTE_2_22=1
-        endif
-    endif
+	# We're a glibc system... Find the actual libc used by our TC, and check if it's >= 2.22...
+	# We only care about that on non-emulated builds (i.e., arm), because compat symbols may be pinned
+	# to different versions on different arches, and we only account for arm in our patches.
+	ifndef EMULATE_READER
+		GLIBC_FILE=$(basename $(notdir $(realpath $(shell PATH='$(PATH)' $(CC) -print-file-name=libc.so.6))))
+		GLIBC_VER=$(shell echo $(GLIBC_FILE) | cut -f2 -d'-')
+		GLIBC_2_22_SORT=$(shell echo -e "$(GLIBC_VER)\n2.22" | sort -V | head -n1)
+		ifeq ($(GLIBC_2_22_SORT), 2.22)
+			ARM_GLIBC_GTE_2_22=1
+		endif
+	endif
 endif
 endif
 endif
 
 # this will create a path named build/arm-none-linux-gnueabi or similar
 MACHINE?=$(shell PATH='$(PATH)' $(CC) -dumpmachine 2>/dev/null)
 ifdef KODEBUG
-    MACHINE:=$(MACHINE)-debug
-    $(info ************ Building for MACHINE: "$(MACHINE)" **********)
-    $(info ************ PATH: "$(PATH)" **********)
-    $(info ************ CHOST: "$(CHOST)" **********)
+	MACHINE:=$(MACHINE)-debug
```

Comment:
I don't see it at a glance, but there must be a stray tab or something somewhere prior to here, unfortunately. :-)

Reply:
Don't understand this one :p 

FYI: I'm checking in vim with

```
:syntax on
:set syntax=whitespace
```
and I see tabs there

Reply:
L269 is the one with spaces, as @frenzie pointed out ;).

Reply:
Don't indent the lines with info and/or move them along these lines:

```diff
diff --git a/Makefile b/Makefile
index 8ab6d7a..3ccca4b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,9 @@
 include Makefile.defs
 
+$(info ************ Building for MACHINE: "$(MACHINE)" **********)
+$(info ************ PATH: "$(PATH)" **********)
+$(info ************ CHOST: "$(CHOST)" **********)
+
 # main target
 all: $(OUTPUT_DIR)/libs $(if $(ANDROID),,$(LUAJIT)) \
                $(if $(USE_LUAJIT_LIB),$(LUAJIT_LIB),) \
diff --git a/Makefile.defs b/Makefile.defs
index b6f72ab..b88840b 100644
--- a/Makefile.defs
+++ b/Makefile.defs
@@ -460,9 +460,7 @@ endif
 MACHINE?=$(shell PATH='$(PATH)' $(CC) -dumpmachine 2>/dev/null)
 ifdef KODEBUG
        MACHINE:=$(MACHINE)-debug
-       $(info ************ Building for MACHINE: "$(MACHINE)" **********)
-       $(info ************ PATH: "$(PATH)" **********)
-       $(info ************ CHOST: "$(CHOST)" **********)
 endif
 OUTPUT_DIR?=build/$(MACHINE)
```

I'm inclined to think this little bit of crucial info shouldn't hide behind special debug flags anyway.

It might need to be done a touch more intelligently though.